kurianc-farmington-ct.md

```diff
@@ -0,0 +1,12 @@
+### Farmington CT
+
+## Places to eat
+
+ - Wood & Tap
+ - Yume
+ - Apricots
+ - Cugino's
+ - Truffles
+
+
+# Places to visit
```

Comment:
Hi, could you make the header format of 'Places to Visit' the same as 'Places to eat', meaning '##' instead of '#'.
Thnx.